scripts/variantstore_wdl/ImportArrayManifest.wdl

```diff
@@ -0,0 +1,150 @@
+version 1.0
+
+workflow ImportArrayManifest {
+
+  input {
+    File extended_manifest_csv
+    File manifest_schema_json
+    String project_id
+    String dataset_name
+    String? table_name
+ 
+    Int? preemptible_tries
+    String? docker
+  }
+
+  String docker_final = select_first([docker, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
```

Comment:
Are you overriding this?  Or are we always testing with this static GATK version?

Reply:
I'm not overriding it in this case because we're not using any GATK tools in this WDL. But maybe we should still be testing the current branch regardless? I know we'll need to for future WDLs (Ingest, calculate metrics, and extract will all use GATK tools that will need to be the docker from the current branch)